lib/ProductOpener/Config_off.pm

```diff
@@ -350,8 +350,13 @@ my @icons = (
 my @related_applications = (
 	{ 'platform' => 'play', 'id' => 'org.openfoodfacts.scanner', 'url' => 'https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner' },
 	{ 'platform' => 'ios', 'id' => 'id588797948', 'url' => 'https://itunes.apple.com/app/id588797948' },
+	{ 'platform' => 'windows', 'id' => '9nblggh0dkqr', 'url' => 'https://www.microsoft.com/p/openfoodfacts/9nblggh0dkqr' },
 );
 
+<a href="https://itunes.apple.com/app/open-food-facts/id588797948">iPhone</a> ou
```

Comment:
Probably a copy & passte mistake, but can you remove the HTML in here, please? 🙂